### DISCUSSION TITLE
grapesjs-plugin-boilerplate, grapesjs-plugin-ckeditor etc. not updated for years, any suggestion?

### DISCUSSION THREAD

**Discussion by wyxcoder** [GrapesJS/grapesjs#4982] [General]
Isn't grapesjs-plugin-boilerplate supported anymore? 
How do you create component now?

**Comment by artf** [comment]
Using the [CLI](https://github.com/GrapesJS/cli) is the proper way to create [plugins](https://grapesjs.com/docs/modules/Plugins.html) (grapesjs-plugin-boilerplate indeed was deprecated years ago)
